package.json

```diff
@@ -55,7 +56,9 @@
     "arrify": "^2.0.1",
     "big.js": "^6.0.0",
     "duplexify": "^4.0.0",
+    "execa": "^5.0.0",
     "extend": "^3.0.2",
+    "gaxios": "^6.0.3",
```

Comment:
should these be dev dependencies instead of regular ones?

Reply:
ah good catch, I forgot the `--dev` flag when I added then